server/app-engine/static/vehicle-assignments.js

```diff
@@ -653,7 +655,7 @@ function handleCloseButtonPress() {
             }
         }
 
-        for (var i=currentAssignments.length-1; i>=0; i--) {
+        for (i=currentAssignments.length-1; i>=0; i--) {
```

Comment:
Ditto. Please see https://stackoverflow.com/questions/5717126/var-or-no-var-in-javascripts-for-in-loop

Reply:
good catch - simply missed this one and a few others in my find/replace